### PR TITLE
OLS-419: Better error messages for auth. endpoint

### PR DESCRIPTION
**Pull request by tisnik**
## Description

Better error messages for auth. endpoint

## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [x] Documentation Update
- [x] Configuration Update
- [ ] Bump-up dependent library

## Related Tickets & Documents

- Related Issue #[OLS-419](https://issues.redhat.com//browse/OLS-419)

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] PR has passed all pre-merge test jobs.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

File: ols/app/endpoints/authorized.py [hunk 1]
```diff
@@ -5,15 +5,39 @@
...
+    200: {
+        "description": "The user is logged-in and authorized to access OLS",
+        "model": AuthorizationResponse,
+    },
+    401: {
+        "description": "Missing or invalid credentials provided by client",
+        "model": UnauthorizedResponse,
+    },
+    403: {
+        "description": "Authorization failure",
```
Comment (bparees) [inline ols/app/endpoints/authorized.py:30]:
```suggestion
        "description": "User is not authorized",
```

File: ols/app/endpoints/authorized.py [hunk 2]
```diff
@@ -5,15 +5,39 @@
...
+        "model": ForbiddenResponse,
+    },
+    500: {
+        "description": "Unexpected error during token review",
+        "model": ErrorResponse,
+    },
+}
+
+
+@router.post("/authorized", responses=authorized_responses)
```
Comment (bparees) [inline ols/app/endpoints/authorized.py:40]: does this affect behavior, or just the openapi schema generation?

Comment (tisnik) [reply ols/app/endpoints/authorized.py:40]: it affects schema and also Swagger UI (http://localhost:8080/docs), nothing else

**Comment by bparees** [comment]
seems safe enough to merge w/o e2e-cluster

**Pull request merged**